projects.go

```diff
@@ -282,6 +282,35 @@ func (s *ProjectsService) ListProjectsUsers(pid interface{}, opt *ListProjectUse
 	return p, resp, err
 }
 
+// ProjectLanguages is a map of strings because the response is arbitrary
+// Gitlab API docs with example: https://docs.gitlab.com/ce/api/projects.html#languages
```

Comment:
Please put a blank comment line in between these lines and call it just `Gitlab API docs:`:

```go
// ProjectLanguages is a map of strings because the response is arbitrary
//
// Gitlab API docs: https://docs.gitlab.com/ce/api/projects.html#languages
```